processor/servicegraphprocessor/config_test.go

```diff
@@ -50,4 +52,28 @@ func TestLoadConfig(t *testing.T) {
 		},
 		cfg.Processors[component.NewID(typeStr)],
 	)
+
+	// Need to set this gate to load connector configs
+	require.NoError(t, featuregate.GlobalRegistry().Set("service.connectors", true))
+	defer func() {
+		require.NoError(t, featuregate.GlobalRegistry().Set("service.connectors", false))
+	}()
+
+	cfg, err = otelcoltest.LoadConfigAndValidate(filepath.Join("testdata", "service-graph-connector-config.yaml"), factories)
```

Comment:
Let's use `confmaptest` to load the component config, so then we can have only relevant to the component configuration in `testdata`.

Reply:
This test was copied from the one above. Do you think we should update both in this PR, or should this be a separate PR?

Reply:
Imho, probably a separate PR would be better.

Reply:
Alright, I'll leave it as is then.